docs/how-to/work-with-pinning-services.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Work with pinning services
+description: Learn how to use or create remote pinning services with IPFS, the InterPlanetary File System.
+---
+
+# Work with remote pinning services
+
+Depending on how you use IPFS, you might find it helpful to use a **remote pinning service** instead of, or in addition to, pinning files on your local IPFS node. Whether it happens remotely or locally, **pinning** an item in IPFS identifies it as something you always wish to keep available, exempting it from the routine "garbage collection" that IPFS does on infrequently-used items in order to efficiently manage storage space. (For more details on pinning as a concept, [see this guide](/how-to/pin-files).)
```

Comment:
```suggestion
Depending on how you use IPFS, you might find it helpful to use a **remote pinning service** instead of, or in addition to, pinning files on your local IPFS node. Whether it happens remotely or locally, **pinning** an item in IPFS identifies it as something you always wish to keep available, exempting it from the routine _garbage collection_ that IPFS does on infrequently-used items in order to efficiently manage storage space. [Learn more about pinning →](/how-to/pin-files).
```